app/controllers/profiles_controller.rb

```diff
@@ -38,11 +42,6 @@ def changed_email?
     user_params[:email].present?
   end
 
-  def set_mail_changes
-    flash[:notice] = I18n.t('flash.notice.profile_email_updated')
-    force_send_confirmation_email if @user.email.blank?
```

Comment:
Maybe, but it's not clear. According to [authentications_controller](https://github.com/call4paperz/call4paperz/blob/master/app/controllers/authentications_controller.rb#L32), we are not accepting new registrations through Twitter.